.env.dist

```diff
@@ -12,3 +12,5 @@ REVISION_DUMMY_API=f0fdadac9163b26232f7b24cc77656ce1417c20d
 REVISION_PATTERN_LIBRARY=0b0336f50ba7385a9c111c1b80a6dc4ba8ac6d38
 # TODO: put this to latest as a default, when `latest` is available
 REVISION_CONTENT_STORE=6c7503408ba51f68d03ededbbac4b13d0e7001f0
+# TODO: put this to latest as a default, when `latest` is available
```

Comment:
This might be too much but, could we remove these TODOs and create tickets/issues for these as a reminder?

Reply:
Yes, created https://github.com/libero/libero/issues/123 (the TODO should bug us to pick that up)